Installing-Oppia-(Linux;-Python-3).md

```diff
@@ -111,7 +111,7 @@ For your virtual environment, we recommend you use [pyenv](https://github.com/py
    ...
    ```
 
-   If you see the warning at the end, add the following lines to your `.bashrc` (see [here](https://askubuntu.com/a/127059) for where to find this file):
+   If you see the warning at the end, add the following lines to your `.bashrc` or `.zshrc` if you are using **Zsh** (see [here](https://askubuntu.com/a/127059) for where to find these files):
```

Comment:
The askubuntu link only refers to `.bashrc`, so maybe instead, let's just replace `.bashrc` with `~/.bashrc` and `.zshrc` with `~/.zshrc` to make it clear where these files are